manim/mobject/svg/svg_mobject.py

```diff
@@ -559,93 +562,140 @@ def generate_points(self):
             )
         )
         # Which mobject should new points be added to
-        self = self
+        prev_command = None
         for command, coord_string in pairs:
-            self.handle_command(command, coord_string)
+            self.handle_command(command, coord_string, prev_command)
+            prev_command = command.upper()
         # people treat y-coordinate differently
         self.rotate(np.pi, RIGHT, about_point=ORIGIN)
 
-    def handle_command(self, command, coord_string):
+    def handle_command(self, command, coord_string, prev_command):
         """Core logic for handling each of the various path commands."""
-        isLower = command.islower()
+        # Relative SVG commands are specified as lowercase letters
+        is_relative = command.islower()
```

Comment:
I don't know anything about SVG syntax BUT: this line indicates that the command being lowercase actually means something. However, L568 is converting every command to uppercase. Can you comment on this discrepancy please?

Reply:
I see. The idea is to collapse the absolute and relative commands in just the type of command, because the logic that relies upon the `prev_command` variable only depends upon the type of command, not whether it's relative.

Instead of capitalizing here, maybe it makes more sense to capitalize within handle_command? Then we don't end up in a weird situation where one of the arguments is capitalized already and the other isn't.

Reply:
Thanks for the explanation. I am neutral about where the lowercase-ing happens, as long as we include a big obvious comment as to why we need to lowercase.